fendermint/app/src/main.rs

```diff
@@ -1,119 +1,51 @@
 // Copyright 2022-2024 Protocol Labs
 // SPDX-License-Identifier: Apache-2.0, MIT
 
-use anyhow::{anyhow, Context};
-use std::path::PathBuf;
-use std::str::FromStr;
-use std::sync::Arc;
-use trace4rs::config::{AppenderId, Policy, Target};
-use trace4rs::{
-    config::{self, Config, Format},
-    Handle,
-};
-
 pub use fendermint_app_options as options;
 pub use fendermint_app_settings as settings;
-use fendermint_app_settings::expand_tilde;
 
 mod cmd;
 
+fn init_tracing(opts: &options::Options) {
+    match &opts.log_dir {
+        Some(log_dir) => {
+            let file = tracing_appender::rolling::never(log_dir, "fendermint.log");
```

Comment:
What does it mean that this is never rolling? Is the sidecar supposed to take car of rolling files? Maybe add a comment on how this will not fill the disk with an ever expanding log?

Reply:
I'll update with rolling logs!